packages/core/src/bags/redux/__tests__/reducer.test.js

```diff
@@ -265,6 +283,59 @@ describe('bags redux reducer', () => {
     });
   });
 
+  describe('bagOperations() reducer', () => {
+    it('should return the initial state', () => {
+      const state = reducer().bagOperations;
+
+      expect(state).toEqual(initialState.bagOperations);
+      expect(state).toEqual({ error: {}, isLoading: {} });
+    });
+
+    it('should handle GET_BAG_OPERATION_REQUEST action type', () => {
+      expect(
+        reducer(undefined, {
+          type: actionTypes.GET_BAG_OPERATION_REQUEST,
+          payload: { bagOperationId: mockBagOperationId },
+        }).bagOperations,
+      ).toEqual({
+        error: { [mockBagOperationId]: null },
+        isLoading: { [mockBagOperationId]: true },
+      });
+    });
+
+    it('should handle GET_BAG_OPERATION_FAILURE action type', () => {
+      expect(
+        reducer(undefined, {
```

Comment:
This should use a state whose `isLoading` value is true to make sure the test can detect errors with the reducer logic instead of undefined, since it should always return false in that case.

Reply:
Yes, exactly. Imagine there is a bug with the implementation of the reducer which makes it always return the same state it was passed in when it is not undefined and the initial state when it is undefined. In that case, the bug will not be caught by this test because it will test the undefined state implementation and not the one for when the `isLoading` is true (This happened before when a case was missing a return in a switch statement).

Reply:
I agree. Changed.